web/src/main/webapp/resources/js/atsy-settings.js

```diff
@@ -8,10 +8,16 @@ function SettingsForm() {
             }
         }
 
-        var form = container.find('form'),
+        var tabs = $('[role="tab"]'),
+            form = container.find('form'),
             table = container.find('table'),
             addAction = container.find('.add');
 
+        tabs.on('shown.bs.tab', function (e) {
+          var tabContentName = e.target.toString().split("#")[1];
+          $('#' + tabContentName + " input[class='name form-control']").focus();
```

Comment:
If you include the hash character in the `tabContentName` above, then you can get rid off the first string concatenation.